html/pfappserver/lib/pfappserver/Base/Form.pm

```diff
@@ -171,6 +176,47 @@ sub id_validator {
    };   
 }
 
+before 'process' => sub {
+    my ($self) = @_;
+    foreach my $field (@{$self->fields}){
+        # Populate all the FingerbankSelect fields with the proper values extracted from the database.
+        if($field->type eq "FingerbankSelect"){
```

Comment:
Why not move this logic to pfappserver::Form::Field::FingerbankSelect?
It does not seem like the correct place for this here.